src/components/ChatLog.js

```diff
@@ -0,0 +1,22 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ChatLog = ({ entries, updateLikes }) => {
+  const ChatEntryComponents = entries.map((entry) => (
+    <ChatEntry
+      // key={entry.id}
+      sender={entry.sender}
+      body={entry.body}
+      timeStamp={entry.timeStamp}
+      updateLikes={updateLikes}/>
+  ));
+  return <div className="chat-log">{ChatEntryComponents}</div>;
+};
+
+ChatLog.propTypes = {
+  entries: PropTypes.array.isRequired,
+};
```

Comment:
You may want to add `updateLikes` as part of the propTypes.

Reply:
`entries` can also be expanded here to add a layer of validation for our props. In this case, we know that `entries` is a list of objects and can further describe the data structure using `arrayOf()` and `shape()`.

```python
ChatLog.propTypes = { 
    entries: PropTypes.arrayOf(PropTypes.shape({ 
        id: PropTypes.number.isRequired, 
        sender: PropTypes.string.isRequired, 
        body: PropTypes.string.isRequired, 
        timeStamp: PropTypes.string.isRequired, 
        liked: PropTypes.bool })).isRequired, 
    updateLikes: PropTypes.func.isRequired };
```